forms-flow-idm/keycloak/themes/README.md

```diff
@@ -0,0 +1,105 @@
+# Apply theme changes
+Existing keycloak setup using Openshift StafulSet `keycloak`.
+
+Needs to add configMap of required file and attache it as volume to exiting pod.
+>It will terminate and recreate pod, due to this keycloak service will be down for 5-10minutes.
+
+### Steps to add configMaps
+In below example consider bcgov theme for the keycloak login page.
+
+__Step 1__ : Login in to oc cli.
```

Comment:
I don't think a VPN is required. I usually connect without it.

Reply:
In my case, it didn’t allow me to login and request end with timeout error message. Once connect with VPN it complete. If its not VPN then I guess it check geolocation and allow only from canada.
Let me add more details in this point that help user.